L1Trigger/L1TMuon/plugins/L1TMuonProducer.cc

```diff
@@ -21,6 +21,7 @@
 // system include files
 #include <memory>
 #include <fstream>
+#include <sstream>
```

Comment:
Hi @rekovic ,
this include could be removed as well.

Reply:
Yes. Thx.

Reply:
I meant the sstream, actually.

Reply:
fstream is needed for std::ofstream m_debugOut;